data/json/recipes/practice/proficiencies.json

```
@@ -0,0 +1,40 @@
+[
+  {
+    "id": "prac_lockpicking_crude",
+    "type": "practice",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_PROFICIENCIES",
+    "name": "lockpicking (improvised)",
+    "description": "Practice picking simple locks",
+    "skill_used": "traps",
+    "practice_data": { "min_difficulty": 1, "max_difficulty": 1, "skill_limit": 1 },
```

```suggestion
    "practice_data": { "min_difficulty": 1, "max_difficulty": 2, "skill_limit": 3 },
```
I think it's reasonable that this continues to train your skill, lockpicking is still part of the devices skill